exercises/grep/description.md

```diff
@@ -13,10 +13,9 @@ The `grep` command takes three arguments:
 3. One or more files in which to search for matching lines.
 
 Your task is to implement the `grep` function, which should read the contents
-of the specified files, find the lines that match the specified pattern
-and then output those lines as a single string. Note that the lines should
-be output in the order in which they were found, with the first matching line
-in the first file being output first.
+of the specified files and find the lines that match the specified pattern.
+The lines should be in the order in which they were found, with the first
+matching line in the first file being first.
```

Comment:
> with the first matching line in the first file being first

This could be clearer. It doesn't say anything about subsequent files or subsequent lines!